guides/README.md

```diff
@@ -1,43 +1,135 @@
 # Foreword
 
-Welcome to the beta version of the MeiliSearch API documentation 🐣
+This guide is made to give you a fast overview of MeiliSearch. Every bit of information is linked to its advanced documentation.
 
-This is our first draft guide in the limbo of the HTTP MeiliSearch routes.
-You can navigate into the documentation using the sidebar or by using the search bar above.
-
-If you spot any typo or any error in the documentation like a miss-documented response body for example,
-please contact us [by email](mailto:bonjour@meilisearch.com) or using the little chat box at the bottom right of this page.
-
-Thank you for your interest and have fun with your HTTP client 🌍
+If you want an even quicker overview we suggest [you look into our quickstart](/tutorials).
 ## Getting Started
 
-MeiliSearch has been developed to provide an easily integrated search solution. Each step of the implementation process has been designed to be as simple as possible. This guide will help you get started with MeiliSearch.
+MeiliSearch has been developed to provide an easily integrated search solution. Each step of the implementation process has been designed to be **as simple as possible**. This guide will help you get started with MeiliSearch.
 
 Contents :
-- Launching an instance of MeiliSearch
-- Create your index and add documents
-- Integrate search
-- Doing Research
+- [Launching an instance of MeiliSearch](/guides/#download-and-launch)
+- [Create your index](/guides/#create-your-index)
+- [Add documents](/guides/#add-documents)
+- [Search !](/guides/#searches)
+
+### Download and launch
 
 First of all, you must have access to a running instance of MeiliSearch.
 
-### Download and launch
 There are [several download possibilities](/guides/advanced_guides/binary.md#download-and-launch).
 
-Once it is downloaded, we need to start our instance of MeiliSearch. The port, as well as the master key, can be defined at launch. There are other [options available](/guides/advanced_guides/binary.md#usage).
+:::: tabs
+
+::: tab cURL
+Download the **latest stable release** of MeiliSearch with **curl**.
 
-One way of installing MeiliSearch is curl :
+Launch the MeiliSearch package to start MeiliSearch's server.
```

Comment:
Isn't it 'binary'?

Reply:
In this context is it from the APT installation, which is a package no ?

Reply:
I mean, we don't 'launch' a package. We 'download' a package, and 'launch' a binary.

Reply:
The binary starts the server.

Reply:
But if everybody agrees with you, go for 'package' :)